test/Tests.Infrastructure/RequiresMsSqlRetryTheoryAttribute.cs

```diff
@@ -6,19 +6,34 @@ namespace Tests.Infrastructure;
 
 public class RequiresMsSqlRetryTheoryAttribute : RetryTheoryAttribute
 {
+    public RequiresMsSqlRetryTheoryAttribute()
+    {
```

Comment:
where this is used?

Reply:
I added this so in case we will want to pass it to `MultiTheoryAttribute` it will compile since it need default ctor, so `Activator.CreateInstance(t)` won't throw.